lib/AST/ASTContext.cpp

```diff
@@ -963,6 +967,11 @@ CanType ASTContext::getAnyObjectType() const {
   return getImpl().AnyObjectType;
 }
 
+CanType ASTContext::getAnyObjectType() const {
```

Comment:
Should this similarly be split up into constraint/existential versions, or do we not need the former anywhere?